sherlock/document.py

```diff
@@ -217,6 +302,26 @@ def from_dict(cls, doc: "Document", dct: Dict[str, Any]) -> "Relation":
 
 @dataclass(frozen=True)
 class Event:
+    """
+    Representation for Events within a Document.
+
+    Saves a single Event.
+    TODO: this is not consistent with Entity:
```

Comment:
see above

Reply:
Same as above, should there be a Class such as Entity and then EntityMention?